client/core/bookie.go

```diff
@@ -194,7 +194,11 @@ func (dc *dexConnection) syncBook(base, quote uint32) (*BookFeed, error) {
 			return nil, err
 		}
 
-		booky = newBookie(dc.log.SubLogger(mktID), func() { dc.stopBook(base, quote) })
+		booky = newBookie(dc.log.SubLogger(mktID), func() {
+			if _, found := dc.books[mktID]; found {
+				dc.stopBook(base, quote)
+			}
+		})
```

Comment:
`stopBook` already does the check, and importantly it locks `dc.booksMtx`, which isn't happening with this new map access here.  So I believe this change isn't needed.